internal/provider/clickhouse_cluster_resource.go

```diff
@@ -351,6 +354,14 @@ func (r *ClickhouseClusterResource) Schema(ctx context.Context, req resource.Sch
 								Optional:            true,
 								MarkdownDescription: "ID of the computational resources preset available to a host (CPU, memory, etc.)",
```

Comment:
We need to implement and use `planmodifier.Int64{&suppressAutoscaledResourcePresetDiff{}}` here.

Reply:
No, validator forbids to set ResourcePresetId along with autoscaler limits, so this sdiff would not be generated, actual preset is ignored in such case